build/build_edge.sh

```diff
@@ -307,7 +308,8 @@ fi
 if [ -f $SETTINGS_FILE ]; then
     source $SETTINGS_FILE
 fi
-source $PETALINUX/settings.sh 
+source $PETALINUX/settings.sh
+source /proj/xbuilds/2023.2_daily_latest/installs/lin64/Vitis/2023.2/settings64.sh
```

Comment:
Perhaps this should be moved to a separate file like vitis.build similar to petalinux.build?
this way the script just need to refer to the vitis.build and we can change the build version in a separate file.